isubit/luggage_projects
# Lugg 551 - Add Alternate URL

Changelog:
- Added Alternate URL field and adjusted the weights of the other project fields accordingly in the content creation form.
- Added Alternate URL and related settings to the Project view.
- Added Alternate URL to full content, search index, and search result displays.

Testing:
- Pull this branch
- Create a project with an alternate url
- Confirm that newly-created project has alternate url in the full-text display
- Search for content. Confirm that alternate url appears in search results
- In the project view, confirm that title now leads to alternate url
- Create a project without an alternate url
- Confirm that the title now leads to the project node
